pkg/clustermanager/cluster_manager.go

```diff
@@ -1110,6 +1165,34 @@ func (c *ClusterManager) CreateEKSAResources(ctx context.Context, cluster *types
 	return c.ApplyBundles(ctx, clusterSpec, cluster)
 }
 
+// UpdateEKSAResources applies the eks-a cluster specs (cluster, datacenterconfig, machine configs, etc.), as well as the
+// release bundle to the cluster. For Kindless management cluster upgrade task, we don't need to pause the cluserSpec and
+// datacenterConfig webhook validation.
+func (c *ClusterManager) UpdateEKSAResources(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec,
+	datacenterConfig providers.DatacenterConfig, machineConfigs []providers.MachineConfig,
+) error {
+	if clusterSpec.Cluster.Namespace != "" {
+		if err := c.clusterClient.CreateNamespaceIfNotPresent(ctx, cluster.KubeconfigFile, clusterSpec.Cluster.Namespace); err != nil {
```

Comment:
I don't think we should do this
This should up to the user to do
If you want, we can have a validation that runs before any of this. That said, this is for updates right? so the namespace has to exist already